src/config/gobjects.rs

```diff
@@ -250,6 +253,10 @@ fn parse_object(toml_object: &Value, concurrency: library::Concurrency) -> GObje
         .lookup("use_boxed_functions")
         .and_then(|v| v.as_bool())
         .unwrap_or(false);
+    let generate_display_trait = toml_object
+        .lookup("generate_display_trait")
+        .and_then(|v| v.as_bool())
+        .unwrap_or(true);
```

Comment:
@GuillaumeGomez Why does this default to true still?

Reply:
By "default" we want to generate the `Display` trait. However, if we have an invalid value, would you prefer instead?

Reply:
I would prefer for it to be opt-in

Reply:
@EPashkin opened #668. You might want to take a look at it. :)